docs/markdown/fallback-wraptool.md

```diff
@@ -20,6 +18,8 @@ To install a wrap, go to your source root, make sure that the `subprojects` dire
 
 This will stage the subproject ready to use. If you have multiple subprojects you need to download them all manually.
 
+*Note* The tool does not work correctly in very old versions of meson (older 0.32.0), you may need to delete the `foo.wrap` file to work aroudn this issue.
+
```

Comment:
"aroudn" and also this chapter is not wrapped to 80 columns.

Reply:
>(older 0.32.0)

older **than** 0.32.0 ??

but this tool first appeared in release 0.32.0.